v

kyle-sutherland opened the pull request:
v

kyle-sutherland committed:
Please enter the commit message for your changes. Lines starting
with '#' will be ignored, and an empty message aborts the commit.

On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
	modified:   netlify.toml

Changes not staged for commit:
	modified:   package-lock.json
	modified:   package.json
	modified:   pages/contact.js
	modified:   styles/globals.css

kyle-sutherland committed:
Please enter the commit message for your changes. Lines starting
with '#' will be ignored, and an empty message aborts the commit.

On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
	modified:   netlify.toml
	modified:   package-lock.json
	modified:   package.json
	modified:   pages/contact.js
	modified:   styles/globals.css

kyle-sutherland committed:
Please enter the commit message for your changes. Lines starting
with '#' will be ignored, and an empty message aborts the commit.

On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
	modified:   cypress/e2e/basic.cy.js
	modified:   netlify.toml
	modified:   package-lock.json
	modified:   package.json

kyle-sutherland committed:
Please enter the commit message for your changes. Lines starting
with '#' will be ignored, and an empty message aborts the commit.

On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
	modified:   styles/globals.css

Changes not staged for commit:
	modified:   pages/_app.js

kyle-sutherland committed:
Changes to be committed:
	modified:   pages/contact.js

kyle-sutherland committed:
Merge branch 'main' of https://github.com/velveteen-rever/KarenSutherlandNextSite

kyle-sutherland committed:
Revert "Please enter the commit message for your changes. Lines starting"

This reverts commit 9f869daa8909b09b806201455aa3e9f1065dbf32.

kyle-sutherland committed:
Merge branch 'main' of https://github.com/velveteen-rever/KarenSutherlandNextSite

kyle-sutherland committed:
d

velveteen-rever commented:
### <span aria-hidden="true">👷</span> Deploy Preview for *sparkly-taiyaki-050e29* processing.


|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 29ba712862d11f39496ad993812ca1bb13d9ec81 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/sparkly-taiyaki-050e29/deploys/64864a32d3bbf6000844fe08 |

kyle-sutherland committed:
Please enter the commit message for your changes. Lines starting
with '#' will be ignored, and an empty message aborts the commit.

On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
	modified:   pages/contact.js

kyle-sutherland committed:
Please enter the commit message for your changes. Lines starting
with '#' will be ignored, and an empty message aborts the commit.

On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
	modified:   cypress/e2e/basic.cy.js
	modified:   netlify.toml
	modified:   package-lock.json
	modified:   package.json
	modified:   pages/contact.js

kyle-sutherland committed:
Changes to be committed:
	modified:   pages/contact.js

kyle-sutherland committed:
alpha1 initial. from last working commit

kyle-sutherland committed:
alpha1 initial. from last working commit

kyle-sutherland committed:
Merge branch 'alpha1'

kyle-sutherland committed:
alpha1 initial. from last working commit
